templates/quickstart-jira-dc-with-vpc.template.yaml

```diff
@@ -169,7 +169,7 @@ Metadata:
       JvmHeapOverride:
         default: JVM Heap Size Override
       KeyPairName:
-        default: Key Name *
+        default: SSH Key-Pair Name (optional)
```

Comment:
/nit `Key Pair`? https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html

I don't mind any of the versions, I just think we should be consistent.

Reply:
Fair point, I'll update them all.